.github/workflows/pythonlinters.yml

```diff
@@ -55,6 +55,10 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install poetry
+          # Poetry complains because antsibull-changelog has dropped Python 3.8
+          # while antsibull still supports it.
+          # TODO: Remove once antsibull stops supporting Python 3.8.
+          sed -i -e 's/^python = .*/python = "^${{ matrix.python-version }}"/' pyproject.toml
```

Comment:
Shouldn't there be a `ref:` at the antsibull-changelog checkout step instead?

Reply:
`ref: 0.17.0`?